autonomy/chain/mint.py

```diff
@@ -132,6 +132,19 @@ def get_contract(public_id: PublicId) -> Contract:
     return Contract.from_dir(directory=contract_dir)
 
 
+def transact(ledger_api: LedgerApi, crypto: Crypto, tx: Dict) -> Dict:
+    """Make a transaction and return a receipt"""
+
+    signed_tx = ledger_api.api.eth.account.sign_transaction(
+        tx,
+        private_key=crypto.private_key,
+    )
```

Comment:
Please use the higher level APIs so it's immediately plugin compatible: https://github.com/valory-xyz/open-aea/blob/main/aea/crypto/wallet.py#L150-L187